### DISCUSSION TITLE
How to use kyuubi Restful API?

### DISCUSSION THREAD

**Discussion by zhichaoleo** [apache/kyuubi#2677] [Q&A] [answered]
Hi Team,

Thanks for your great work! I found that the issue https://github.com/apache/incubator-kyuubi/issues/950 has been closed. 
And based on the dicussion: https://github.com/apache/incubator-kyuubi/discussions/917, does the feature has been released with the version v1.5.1?
We are looking forward to use the rest api feature. Could you share some knowledge about how to use kyuubi's restful api?

BR & Thanks,
Alex

**Comment by yaooqinn** [comment, marked as answer]
The upcoming https://github.com/apache/incubator-kyuubi/issues/2652 will provide a UI on it and https://github.com/apache/incubator-kyuubi/issues/2240 will supports a programing rest client to support run spark jar jobs via rest

**Reply by zhichaoleo** [reply]
Hi @yaooqinn ,

Thanks for your response. Looking forward to these exciting features.

BR,
Alex
